marshaller.go

```diff
@@ -12,12 +12,35 @@ import (
 )
 
 const (
-	anyVersion              = math.MaxUint8
-	maxInitialSliceCapacity = 100 * 1024
+	anyVersion       = math.MaxUint8
+	maxSliceCapacity = 64 * 1024
```

Comment:
I think length of some box types can be larger than 64k (ex. `ctts` box).

Reply:
i increased the maximum capacity to 128k